README.md

```diff
@@ -91,6 +91,25 @@ self.postMessage({foo: 'foo'})
 self.addEventListener('message', (event) => { console.log(event); });
 ```
 
+
+## Cross-origin policy workarounds
+

```

Comment:
Can we describe the feature to be less use case specific?

Feel free to add the worker cross-origin policy as an example underneath.

- Add `publicPath` to table that describes the api
- Add `## Public Path` docs below the existing sections describing the api options
- Add generic example
- Add note about workaround for cross-origin specific usecase.

Reply:
@michael-ciniawsky Other than the documentation, it looks fine pending a thumbs up from @TrySound

Reply:
👍  I've rebased & updated the documents.